Zajecia2/O-mnie/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="pl">
```

Comment:
pusciles 2 razy to samo, masz 2 foldery "Zajecia2" z tym samym contentem, skasuj jeden

Reply:
Gdzie mam dokładnie to usunąć? Lokalnie mam tylko jeden folder. W Gitcie widzę tylko to:
![Zrzut ekranu 2024-03-17 o 13 29 29](https://github.com/mart1313/FrontendS01/assets/159488478/c8115016-a07e-4189-bdb9-c31350561e28)
W Sourcetree usunąłem branch. Przy okazji dałem trochę ciała naniosłem zmiany w plikach zajęcia-2 z zadaniem z trzecich zajęć ale nie utworzyłem wcześniej nowego branchu i nie zmergowałem :/ Trochę się jeszcze gubię w tym workflow.